README.md

```diff
@@ -54,32 +54,42 @@ If you're more familiar with deep learning systems like TensorFlow or PyTorch, y
-         |                                 ·           ··                                 
-         |               ·           ·  · ·· ·  ···  ·  ·· ·  ·    · · ·                  
-         |                ·   ·   ··························· ··· ·      ·  ···           
-         |                   · ·············································      ··      
-    2.10 |             ·····   ···············································  ···       
-         |          · ·  ···················································· ····      · 
-         |·   ·  ·   ···························∘··∘·····························  ·      
-         |      ··························∘·∘∘∘∘∘∘∘∘∘∘∘∘∘∘∘∘∘···················· ··      
-         |     ·      ··················∘∘∘∘∘∘∘∘∘○○○○∘○○∘∘∘∘∘∘···················· · ·    
-    2.00 |        ·····················∘∘∘∘∘∘○○○○○○○○○○○○○∘∘∘∘∘∘······················ ·  
-         |           ··················∘∘∘∘∘∘○○○○○○○○○○○○○∘∘∘∘∘∘···················       
-         |         ·····················∘∘∘∘∘∘∘○○○○○∘∘○○∘∘∘∘∘∘∘∘················ ·····    
-         |          ·······················∘∘∘∘∘∘∘∘∘∘∘∘∘∘∘∘······················· ··  ·  
+[info] Running rainier.example.FitNormal
+    2.19 |
+         |                                 ·           ··
+         |               ·           ·  · ·· ·  ···  ·  ·· ·  ·    · · ·
+         |                ·   ·   ··························· ··· ·      ·  ···
+         |                   · ·············································      ··
+    2.10 |             ·····   ···············································  ···
+         |          · ·  ···················································· ····      ·
+         |·   ·  ·   ···························∘··∘·····························  ·
+         |      ··························∘·∘∘∘∘∘∘∘∘∘∘∘∘∘∘∘∘∘···················· ··
+         |     ·      ··················∘∘∘∘∘∘∘∘∘○○○○∘○○∘∘∘∘∘∘···················· · ·
+    2.00 |        ·····················∘∘∘∘∘∘○○○○○○○○○○○○○∘∘∘∘∘∘······················ ·
+         |           ··················∘∘∘∘∘∘○○○○○○○○○○○○○∘∘∘∘∘∘···················
+         |         ·····················∘∘∘∘∘∘∘○○○○○∘∘○○∘∘∘∘∘∘∘∘················ ·····
+         |          ·······················∘∘∘∘∘∘∘∘∘∘∘∘∘∘∘∘······················· ··  ·
          |       ·· ··························································   ·  ·    ·
-    1.90 |          ·  ························································     ·     
-         |                ············································· ···· ·    · ·     
-         |                      ·   · ··· ··················· ···· ·  ·    ·              
-         |                           ··· ·  · ·· ·· ·  ··  · ·· ··                        
-         |                              ·        ··                                       
-    1.81 |                                       ·                                        
```

Comment:
Emacs had some fun deleting trailing whitespace. I can update the PR for a minimal diff, if desired.